app/core/abilities/ability-strategy.ts

```diff
@@ -1,15 +1,16 @@
 import { Transfer } from "../../types"
-import { Ability } from "../../types/enum/Ability"
 import { Effect } from "../../types/enum/Effect"
-import { Item } from "../../types/enum/Item"
 import { Passive } from "../../types/enum/Passive"
 import { Synergy } from "../../types/enum/Synergy"
 import { distanceC } from "../../utils/distance"
 import Board from "../board"
 import { PokemonEntity } from "../pokemon-entity"
 import PokemonState from "../pokemon-state"
-import { AbilityStrategies } from "./abilities"
 import { min } from "../../utils/number"
+import { triggerItems } from "../items"
```

Comment:
`triggerItemsEffects` intended instead?

Reply:
My items file wasn't updated properly. triggerItems is the current name of the routing function